README.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started with Create React App
+# Getting Started with Create React App 
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
```

Comment:
- [ ] Let's make use of the Microverse README [template](https://github.com/microverseinc/readme-template/blob/master/README.md) instead of the generated README from creating a react app. The Microverse README template has some important sections that are required by every student to have. Please consider removing the generated README and using the Microverse Template. Do not forget to personalize the content of the template and remove sections that are not yet implemented in this project. Thank you.